docs/Releasing.md

```diff
@@ -0,0 +1,15 @@
+# Checklist for making a new release
+
+These are the code changes that need to take place
+- [ ] Create and push git tag
+- [ ] Update version in CMakeLists.txt
+- [ ] Update changelog at data/debian/changelog
+- [ ] Update changelog at data/rpm/flameshot.spec
```

Comment:
also need update version at /data/rpm/flameshot.spec

Reply:
Thanks, i updated those comments, ill remove the version in the next release.